src/main/pages/che-7/end-user-guide/assembly_configuring-openshift-oauth.adoc

```diff
@@ -42,7 +42,7 @@ grantMethod: prompt
 ----
 . Add the OpenShift SSL certificate to the {prod-short} Java trust store.
 //TODO yhontyk to uncomment later. For now it turns into an xref that breaks the bccutil build :( 
-//See link:{site-baseurl}che-7/advanced-configuration-options/#adding-custom-certificates-to-trust-store_advanced-configuration-options[Adding custom public SSL certificates to {prod-short} trust-store]
+//See link:{site-baseurl}che-7/advanced-configuration-options/#adding-self-signed-SSL-certificates-to-{prod-short}_advanced-configuration-options[Adding self-signed SSL certificates to {prod-short}]
```

Comment:
Should this still be commented out?

Reply:
Solving this out :)

Reply:
I created an attribute link for the two scenarios from this file. Build it locally in upstream and downstream and it works fine.